### DISCUSSION TITLE
auto generating selectors can't work with persisting store data?

### DISCUSSION THREAD

**Discussion by R0boter** [pmndrs/zustand#2498] [Q&A] [answered]
When I wanted to use the example in #1412 whit persisting store data, react app reported an error "useStore.use.bears() is not a function"
dose the selectors can't work with persisting store?
thanks for your help

**Comment by dai-shi** [comment, marked as answer]
I think you need to `partialize` to avoid serializing `use` property.

**Reply by R0boter** [reply]
thank you for your answer
